obs_status.yml

```diff
@@ -1,4 +1,5 @@
 bad:
+  1081092600: 12
```

Comment:
can you remind me why we need a "manual mag 'bad' label"? If the magnitude was set manually it does not mean the star is bad, and instead the star can be used by the selection algorithm based on the magnitude value/error.

Reply:
Right, if the magnitude was set manually it does not **necessarily** mean the star was bad, but in these cases, I thought the magnitude was set manually because the star was poorly tracked and the ACA reviewer needed to make a judgement call.  I think if the star is poorly tracked enough that we need to make that judgement call, it is likely it can also be marked bad with no real downside.